docs/source/concepts/NexusFile.rst

```diff
@@ -3,18 +3,37 @@
 Nexus File
 ==========
 
-A **Nexus file** is a type of data file used by ISIS traget station 2
+A **Nexus file** is a type of data file used by various instruments
 and by MantidPlot. The format appears to be like an XML file plus some
 unprintable characters.
 
-ManditPlot is capable of saving certain types of
-:ref:`workspace <workspace>` as a Nexus file. It can also save a
+Mantidplot is capable of loading certain types of Nexus files and of saving certain types of
+:ref:`workspace <workspace>` as a Nexus file.  It can also save a
 :ref:`project <project>` as a mantid file plus Nexus files.
 
 Structure
 ---------
 
-To be completed.
+The general structure of NEXUS files is explained in http://download.nexusformat.org/doc/html/user_manual.html .
+
+Here are some specific details:
+
+ISIS uses NEXUS files for both histrogram and event data and SNS uses NEXUS for event data only. 
+Also both ISIS and SNS use the same structure for event data. 
+Hence there are two principal types of NEXUS files loaded by Mantid 
+
+ - ISIS histogram Nexus file, which is loaded by :ref:`LoadISISNexus <algm-LoadISISNexus>` and
+ - Event Nexus file, which is loaded by :ref:`LoadEventNexus <algm-LoadEventNexus>`.
+
+ISIS also have two versions of Nexus files for muon histogram data:
+
+ - Muon Nexus file v1, which is loaded by :ref:`LoadMuonNexus v1<algm-LoadMuonNexus-v1>` and
+ - Muon Nexus file v2, which is loaded by :ref:`LoadEventNexus v2<algm-LoadMuonNexus-v2>`.
+
+
+Also there is a kind of Nexus file, which is produced by Mantid, when it saves a workspace
```

Comment:
The word *Also* appears three times in this paragraph. Consider using synonyms like *Additionally*. (Might not be necessary just picked up on it)